src/main/java/com/webaut/project/pages/Login.java

```diff
@@ -1,8 +1,28 @@
 package com.webaut.project.pages;
 
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
 public class Login extends AbstractPage {
 
+    @FindBy(css = "input[id = user]")
+    private WebElement userNameTextField;
+
+    @FindBy(css = "input[id = password]")
+    private WebElement userPasswordTextField;
+
+    @FindBy (css = ".button.button-green")
```

Comment:
Remove the blank space after the annotation FindBy please

Reply:
thanks for the observation, I removed the space